src/Perpetuum/Zones/Terrains/Materials/Plants/PlantHandler.cs

```diff
@@ -34,13 +36,16 @@ public class PlantHandler : IPlantHandler
         private int _areaDoneY;
         private readonly int _areaAmount;
         private bool _stopSignal;
+        private int _total_area;
 
         public Area WorkArea { private get; set; }
 
         public PlantHandler(IZone zone)
         {
             _zone = zone;
             _areaAmount = zone.Size.Width / AREA_SIZE; //the amount of areas
+            _total_area = (zone.Size.Width / AREA_SIZE) * (zone.Size.Height / AREA_SIZE);
+            _plantsTimer = new IntervalTimer(TimeSpan.FromHours(FULL_PLANT_REGEN_PASS / _total_area));
```

Comment:
Better! but int division here will cause issues.  Cast some of those, or declare them, as doubles to avoid getting 0 for every result!